### DISCUSSION TITLE
<aside> content with footnote semantics are no displayed in Thorium 2.1.0

### DISCUSSION THREAD

**Discussion by VijayShankar1** [edrlab/thorium-reader#1811] [General]
If we captured the table footnotes in `<aside>` tag, than it is not viewable in Thorium 2.1.0. However it is viewable in Thorium version 1.8.0. Is this a bug in Thorium 2.1.0.

We have placed the coding structure for your reference. Thanks!

```
<aside epub:type="footnote" role="doc-footnote" class="footnote" id="tab2_1_1">
<a epub:type="backlink" role="doc-backlink" href="#Ctab2_1_1">i</a> XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX (<a epub:type="biblioref" role="doc-biblioref" id="Cref2_87" href="#ref2_87">XXXX 2019</a>).</aside>
```

**Comment by danielweck** [comment]
I believe Thorium 2.1.0 (the latest version as of the time of writing) implements this functionality correctly. `<aside epub:type="footnote">` means "hide when the EPUB3 popup footnotes feature is activated, or show when deactivated so that a normal hyperlink can jump to it".
Apple iBooks: https://help.apple.com/itc/booksassetguide/en.lproj/itccf8ecf5c8.html
> Note: When adding pop-up footnotes in EPUB 3 books, you can replace the `<aside>` element with a `<div>` or `<p>` element. Use the `<aside>` element when you want to hide the footnote; use a `<div>` or `<p>` element when you want the footnote to appear in the normal reading view. If you use `<div>` or `<p>` and a user clicks the footnote link, the content appears in a popup, but the footnote is also visible as part of the text on the page.
